webapp/src/Controller/API/SubmissionController.php

```diff
@@ -113,7 +113,7 @@ public function singleAction(Request $request, string $id)
      * @return int
      * @Rest\Post("")
      * @OA\Post()
-     * @IsGranted("ROLE_TEAM")
+     * @IsGranted("ROLE_TEAM", message="You need to be a team to add a submission")
```

Comment:
I would reword this as "You need to have the Team Member role to..." so it's more clear what exact action you need to do on the user to fix it.